docs/src/main/paradox/scala/http/server-side/server-https-support.md

```diff
@@ -1,20 +1,24 @@
 # Server-Side HTTPS Support
```

Comment:
Github detects this page as a moved from `java/http/server-side-https-support.md` to the `scala/http/server-side-https-support.md` and not as a symlink. But looking at the files themselves seems good.